hardware/serial/serial.cpp

```diff
@@ -2,8 +2,10 @@
 #include "stdafx.h"
 #include <algorithm>
 
-#if !defined(_WIN32) && !defined(__OpenBSD__) && !defined(__FreeBSD__)
+#ifdef __sun
```

Comment:
I like more:

```
#ifdef __sun
# include <alloca.h>
#endif
#include <stdlib.h>
```
Just in case that SmartOS might want both for some reason.

Reply:
Agreed.